modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -716,7 +716,7 @@ public String signature() {
                         failCommandWithCarError(cm);
                         return idleState;
                     } else if (carState.isBusy()) {
-                        return new CaObserveSenderImpl.WaitApplyBusy(cm, val, carState, observeState);
+                        return new CaObserveSenderImpl.WaitApplyIdle(cm, val, carState, observeState);
```

Comment:
Why did it go to `WaitApplyBusy` if `carState` was already busy? Looks like a hidden bug.

Reply:
👍